force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -300,17 +416,26 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Map<String, Integer> addItemsToMap() {
         // Initialize an empty Map
-
+        Map<String, Integer> fruitMap = new Map<String, Integer>();
         // Add key-value pairs to the Map
         // add Apples => 5
         // add Oranges => 10
         // add Bananas => 15
         // add Pears => 20
         // add Grapes => 25
 
-        return null; // Replace null with the variable you used to store the result
+       // Create a list of fruit names
+       List<String> fruits = new List<String>{'Apples', 'Oranges', 'Bananas', 'Pears', 'Grapes'};
+       // Create a list of corresponding quantities
+       List<Integer> quantities = new List<Integer>{5, 10, 15, 20, 25};
+       // Use a loop to add key-value pairs to the Map
+       for (Integer i = 0; i < fruits.size(); i++) {
```

Comment:
The is an interesting way to do this. Kudos for trying something diffferent to see other ways